packages/flame/lib/src/components/position_component.dart

```diff
@@ -83,6 +85,18 @@ class PositionComponent extends BaseComponent {
     _onModifiedSizeOrAnchor();
   }
 
+  @override
+  @mustCallSuper
+  void update(double dt) {
+    super.update(dt);
+  }
+
+  @override
+  @mustCallSuper
+  void render(Canvas canvas) {
+    super.render(canvas);
+  }
```

Comment:
do you need to override just to call super?

Reply:
This was needed before when `Component` didn't have `@mustCallSuper`, since OxygenGame shouldn't be force to call super, but now since it is using the low-level API I can re-add `@mustCallSuper` on `Component` instead. :)